.github/workflows/ci.yml

```diff
@@ -7,15 +7,21 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v2
       - name: Setup ruby
-        uses: actions/setup-ruby@v1
+        uses: ruby/setup-ruby@v1
```

Comment:
Ohh I see that in the last commit you moved to ruby/setup-ruby

Reply:
Yep!